internal/ignition/ignition.go

```diff
@@ -278,25 +320,38 @@ func (g *installerGenerator) createLsoManifests() error {
 	}
 	return nil
 }
-func (g *installerGenerator) generateLsoManifests(ctx context.Context, installerPath string, envVars []string) error {
-	if g.checkLsoEnabled() {
+
+func (g *installerGenerator) generateOperatorsManifests(ctx context.Context, installerPath string, envVars []string, ocsValidatorConfig *ocs.Config) error {
+	lsoEnabled := g.checkOcsEnabled()
+	ocsEnabled := g.checkLsoEnabled()
+	if lsoEnabled || ocsEnabled {
 		err := g.createManifestDirectory(installerPath, envVars)
 		if err != nil {
-			g.log.Error(err)
+			g.log.Error("Failed to create Manifest directory ", err)
 			return err
 		}
-		err = g.createLsoManifests()
+	}
+
+	if lsoEnabled {
+		err := g.createLsoManifests()
 		if err != nil {
 			g.log.Error(err)
 			return err
 		}
 	}
 
+	if ocsEnabled {
```

Comment:
This will generate OCS manifests even if only LSO is selected.

Reply:
fixed